src/handlers/next/create-manifest-handler.ts

```diff
@@ -1,12 +1,13 @@
 import { NextApiHandler, NextApiRequest } from "next";
 
-import { getBaseUrl } from "../../headers";
+import { getBaseUrl, getSaleorHeaders } from "../../headers";
 import { AppManifest } from "../../types";
 
 export type CreateManifestHandlerOptions = {
   manifestFactory(context: {
     appBaseUrl: string;
     request: NextApiRequest;
+    schemaVersion: number | null;
```

Comment:
Non blocking; you can add comment (jsdoc) mentioning here that null means <3.15 

IDEs are using comments for hints